docs/sources/permissions/teams.md

```diff
@@ -0,0 +1,41 @@
++++
+title = "Teams"
+description = ""
+keywords = ["grafana", "configuration", "documentation", "admin", "users", "teams", "permissions"]
+type = "docs"
+[menu.docs]
+parent = "permissions"
+weight = 20
++++
+
+# Teams
```

Comment:
"Teams" is too vague for this content, the title does not communicate to me that there are tasks. Either rename it to something like "Manage teams" or break up the tasks into separate topics, each with a one-sentence introduction to provide context to the user. 

"Create a team that you can add users to."
"Delete a team from Grafana. This does not delete users assigned to the team."